department-of-veterans-affairs/va.gov-cms
# VACMS-6085: Add fields to banner content type.

## Description
Relates to #6085

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/129092011-a4216b70-a379-41ee-9e30-3372120f6483.png)

## QA steps
 - As an administrator, go to `/admin/structure/types/manage/banner/fields` and visually verify presence of 2 new fields.

### Definition of Done

### Select Team for PR review

### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
